.github/workflows/battery_plus.yaml

```diff
@@ -27,6 +27,27 @@ jobs:
       - name: "Build Example"
         run: ./.github/workflows/scripts/build-examples.sh android ./lib/main.dart
 
+  android_integration_test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v2
+      - name: "Install Flutter"
+        run: ./.github/workflows/scripts/install-flutter.sh stable
+      - name: "Install Tools"
+        run: ./.github/workflows/scripts/install-tools.sh
+      - name: "Bootstrap Workspace"
+        run: melos bootstrap
+      - name: "Android Integration Test"
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 30
+          arch: x86_64
+          target: google_apis
+          profile: Nexus 5X
+          script: ./.github/workflows/scripts/integration-test.sh android battery_example
```

Comment:
Should be `battery_plus_example` as it is the name in the pubspec.yaml here: https://github.com/fluttercommunity/plus_plugins/blob/main/packages/battery_plus/battery_plus/example/pubspec.yaml#L1

Reply:
Got it~